src/java/eu/rssw/pct/oedoc/ClassDocumentationVisitor.java

```diff
@@ -448,4 +469,12 @@ private static String getDataTypeName(TypeName typeName) {
             return typeName.getName();
 
     }
+
+    private String ltrim(String s) {
```

Comment:
The string `   Test Test2` will return `Test2`, not `Test Test2` I'd expect from an ltrim function. Is that expected ?

Reply:
No, it's not the behavior I expect.
I just copy past it from `ClassDocumentationVisitor`.
I will fix it.

Reply:
Right :)